### PR TITLE
Homework9: Andrei Kulbitski

### PR DESCRIPTION
**Pull request by kulbitski-andrei**
Homework 9

### CODE REVIEW

File: homework_9.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+def char_counter(text):
+    """Accepts string, returns string
+    with calculation of amount of every character"""
+    text_improved = ""
+    counter = 0
+    new_letter = True
+    position_index = -1
+
+    # for _ in range(len(text)):
```
Comment (Stml89) [inline homework_9.py:79]: remove it

File: homework_9.py [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+        if text[position_index] != text[position_index + 1]:
+            if counter < 2:
+                counter = ""
+            text_improved += f"{text[position_index]}{counter}"
+            new_letter = True
+            counter = 0
+        else:
+            counter += 1
+
+    return None
```
Comment (Stml89) [inline homework_9.py:103]: return False

**Review by Stml89 — APPROVED** [review]

File: homework_9.py (hunk 2 above)
Comment (kulbitski-andrei) [reply homework_9.py]: Changed to "return False"

File: homework_9.py (hunk 1 above)
Comment (kulbitski-andrei) [reply homework_9.py]: Done!

**Pull request merged**